pom.xml

```diff
@@ -635,6 +635,52 @@
                     <generateGitPropertiesFilename>${project.build.outputDirectory}/git.properties</generateGitPropertiesFilename>
                 </configuration>
             </plugin>
+
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>${exec-maven-plugin.version}</version>
+                <executions>
+                    <execution>
+                        <id>create-licenses</id>
+                        <configuration>
+                            <mainClass>io.confluent.licenses.LicenseFinder</mainClass>
+                            <arguments>
+                                <!-- Note use of development instead of package so we pick up all dependencies.
+                                     This assumes both subprojects will be packaged together. -->
+                                <argument>-i ${project.build.directory}/${project.build.finalName}-package/share/java/acl</argument>
+                                <argument>-o ${project.parent.basedir}/licenses/overrides</argument>
+                                <argument>-f</argument>
+                                <argument>-h ${project.build.directory}/${project.build.finalName}-package/share/doc/${project.name}/licenses.html</argument>
+                                <argument>-l ${project.build.directory}/${project.build.finalName}-package/share/doc/${project.name}/licenses</argument>
+                                <argument>-n ${project.build.directory}/${project.build.finalName}-package/share/doc/${project.name}/notices</argument>
+                                <argument>-t ${project.name}</argument>
+                                <argument>-x connect-api-${ce.kafka.version}.jar,connect-json-${ce.kafka.version}.jar,connect-runtime-${ce.kafka.version}.jar,control-center-${io.confluent.blueway.version}.jar,licenses-${io.confluent.license-file-generator.version}.jar,confluent-licensing-new-${ce.kafka.version}.jar,control-center-frontend-${io.confluent.blueway-fe.version}.jar,confluent-command-${io.confluent.blueway.version}.jar</argument>
+                            </arguments>
+                        </configuration>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>java</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <includeProjectDependencies>true</includeProjectDependencies>
+                    <includePluginDependencies>true</includePluginDependencies>
+                    <executableDependency>
+                        <groupId>io.confluent</groupId>
+                        <artifactId>licenses</artifactId>
+                    </executableDependency>
+                    <mainClass>io.confluent.controlcenter.acl.StartupAcl</mainClass>
```

Comment:
I don't actually remember what this part of the config does, but you'll want to change this to the main class for each module you put this in.